awscli/botocore/client.py

```diff
@@ -295,6 +299,31 @@ def _register_s3_control_events(self, client):
             return
         S3ControlArnParamHandlerv2().register(client.meta.events)
 
+    def _default_s3_presign_to_sigv2(self, signature_version, **kwargs):
```

Comment:
This function was ported from the current state of botocore: https://github.com/boto/botocore/blame/develop/botocore/client.py#L475

And supported by the following PR:

- https://github.com/SamRemis/botocore/commit/5af85bf0b3cf6dac8d0487a2e97425cf3d53e4ee